package.json

```diff
@@ -154,7 +156,7 @@
 		"webpack": "4.41.0",
 		"worker-farm": "1.7.0"
 	},
-	"npmPackageJsonLintConfig": {
+	"npmpackagejsonlint": {
```

Comment:
This one was an undocumented breaking change but explained in https://github.com/tclindner/npm-package-json-lint/issues/140. @ntwb, I see you had the same issue :)

Reply:
Yes, probably should have been added to the latest release notes...

Reply:
Sorry for missing this in the docs. The change log and migration guide have been updated!